push_notifications/services/zeropush.py

```diff
@@ -16,30 +18,39 @@ def __init__(self, settings):
             raise ImproperlyConfigured('For ZeroPush to work you need an '
                                        'AUTH_TOKEN in the configuration')
 
+    @staticmethod
+    def process_expiry(expiry):
+        if isinstance(expiry, datetime):
+            expiry = expiry.second
+
+        if isinstance(expiry, timedelta):
```

Comment:
To allow users to pass either a datetime or a timedelta object. We have to convert it to seconds for `zeropush`. @glemmaPaul can weigh in on why he wanted those options to be available. 

```python
>>> import datetime
>>> from datetime import timedelta
>>> datetime.datetime.now()
datetime.datetime(2015, 5, 4, 17, 32, 55, 990142)

# for datetime objects second works
>>> datetime.datetime.now().second
59

# seconds do not
>>> datetime.datetime.now().seconds
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'datetime.datetime' object has no attribute 'seconds'

# for timedelta objects the opposite is true
>>> timedelta(seconds=10)
datetime.timedelta(0, 10)
>>> timedelta(seconds=10).seconds
10
>>> timedelta(seconds=10).second
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'datetime.timedelta' object has no attribute 'second'
```